src/drafter_private.h

```diff
@@ -0,0 +1,34 @@
+//
+//  drafter_private.h
+//  drafter
+//
+//  Created by Pavan Kumar Sunkara on 20/10/16.
+//  Copyright © 2016 Apiary. All rights reserved.
+//
+
+//TODO: This should be merged with drafter.h in next major release
+#ifndef DRAFTER_PRIVATE_H
+#define DRAFTER_PRIVATE_H
+
+#include "drafter.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct {
+    bool requireBlueprintName;
+} drafter_parse_options;
+
+DRAFTER_API int drafter_parse_blueprint_with_options(const char* source,
+                                                     drafter_result** out,
```

Comment:
I think we should put the options before the output that seems to make more logical sense to me. What do you think?